textproc/tesseract/Portfile

```diff
@@ -25,26 +35,32 @@ maintainers         {mark @markemer} openmaintainer
 
 description         Open source OCR engine
 
-long_description    The Tesseract OCR engine was one of the top  3  engines in \
-                    the 1995 UNLV Accuracy test.  Between 1995 and 2006 it had \
-                    little work done on it,  but it is  probably  one  of  the \
-                    most accurate  open  source  OCR  engines  available.  The \
-                    source code will read a binary, grey or  color  image  and \
-                    output text.  A tiff reader is built  in  that  will  read \
-                    uncompressed TIFF images,  or libtiff can be added to read \
-                    compressed images.
+long_description    This package contains an OCR engine - libtesseract and a \
+                    command line program - tesseract. Tesseract 4 adds a new \
+                    neural net (LSTM) based OCR engine which is focused on line \
+                    recognition, but also still supports the legacy Tesseract \
+                    OCR engine of Tesseract 3 which works by recognizing \
+                    character patterns.
 
 if {${subport} eq ${name}} {
-    revision                2
-
     github.livecheck.regex  {(\d\.\d+(\.\d+)?(?!-rc))}
 } else {
     livecheck.type          none
 }
 
-subport ${name}-training {
-    revision                2
+if {${subport} in [list tesseract tesseract-training]} {
+    notes "To use tesseract you must also install one of\nits language data subports. (ex tesseract-eng)"
```

Comment:
Nit-pick: Don't embed newlines in port notes - at least not to try to wrap the text - as MacPorts base will handle that for you automatically.

Embedded newlines are okay, though, if you're specifically starting a new paragraph. (Or a new bullet item, etc.)